modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/actions.js

```diff
@@ -25,28 +25,35 @@ export const ACTIONS = {
 			Liferay.Language.get('delete-collection'),
 			viewDeleteFragmentCollectionsURL,
 			(selectedItems) => {
-				const fragmentCollectionsForm = document.getElementById(
-					`${portletNamespace}fragmentCollectionsFm`
-				);
+				if (selectedItems.length) {
```

Comment:
I tweaked my code to have a null check and an early return, see https://github.com/pei-jung/liferay-portal/pull/824/commits/c3bfd84d468dddef36403ce175641fd8dec4f48f
- Although unlikely, it is possible that `selectedItems` is unset, if devs opt to use custom event.
- Early return is just to buy space, as Brian would say

Reply:
Sounds good!  Thanks!  I will update my code as necessary.

Reply:
Fixed at https://github.com/liferay-frontend/liferay-portal/pull/948/commits/cf4ea362db69088c0061072bbc85b7e67bc5ba75#diff-d144a92ad201a6b94c6a6cf6c91984dd47a7f4a02ada1724e056497167b90c33R28